uktrade/directory-cms
# GP2-3158: Refactor International Homepage for new content

This changeset radically refactors the CMS page model and serializer for the InternationalHomePage, ready for the new design.

It adds a streamfield using the new InternationalHomepagePanelBlock and a large number of fields which are no longer needed, including some which appear to be from an the previous incarnation of the homepage.

**AS A RESULT, THE MIGRATION IN THIS CHANGESET TAKES A LONG TIME TO RUN**: locally it takes ~22 minutes, but hopefully on the PaaS it will run a lot more quickly.

Note that until great-international-ui is updated to match the new data structure, this will effectively break great-international's homepage, so coordination of releases must be done.

 - [X] Change has a jira ticket that has the correct status.


### How to test:

* Back up your local directory-cms database first
* Locally, pull the branch and run the migration (this will take at least 20 minutes...)
* Edit the Great International homepage in the CMS - confirm you can set a hero title and also a streamfield of 'panels'
* You can publish the page, but if you do so, note that this will break the page on the great-international-ui service, which has yet to be updated to suit the new data structures
